osf/models/preprint_provider.py

```diff
@@ -21,6 +21,7 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     header_text = models.TextField(default='', blank=True)
     description = models.CharField(null=True, blank=True, max_length=256)  # max length on prod: 56
     banner_name = models.CharField(null=True, blank=True, max_length=128)  # max length on prod: 19
+    domain = models.CharField(null=True, blank=True, max_length=253)  #
```

Comment:
I actually had to look this one up; nice catch. For future generations, this was informative:

http://blog.sacaluta.com/2011/12/dns-domain-names-253-or-255-bytesoctets.html